test/utils/getUnhandledPops-test.js

```diff
@@ -0,0 +1,33 @@
+import _ from 'lodash';
+import getUnhandledProps from 'src/utils/getUnhandledProps';
+
+// Helper class that takes in props and merges defaultProps
+class TestClass {
+  constructor(props) {
+    this.props = _.assign({}, this.constructor.defaultProps, props);
+    this.unhandledProps = getUnhandledProps(this);
+  }
+}
+
+describe.only('getUnhandledProps', () => {
+  it('removes props defined in defaultProps', () => {
+    TestClass.defaultProps = {imHandled: 'thanks'};
+    new TestClass()
+      .unhandledProps
+      .should.not.have.any.keys(_.keys(TestClass.defaultProps));
+  });
+  it('removes props defined in propTypes', () => {
+    TestClass.propTypes = {imHandled: 'thanks'};
+    new TestClass()
+      .unhandledProps
+      .should.not.have.any.keys(_.keys(TestClass.defaultProps));
+  });
+  it('leave props not in defaultProps || propTypes in tact', () => {
+    TestClass.defaultProps = {imHandled: 'thanks'};
+    TestClass.propTypes = {alsoHandled: 'got it'};
+    const props = {thisShould: 'still be here'};
```

Comment:
Good question!  We've updated the linter.  If you make a variable whose value never changes, it will tell you to make it a const instead.  The only use case for a `let` now is for a variable whose value changes.  Example:

```
let hasCertainChild;
React.Children.forEach(child => {
  if (someCondition(child) {
    hasSomeChild = true;
  }
})
```

In this case, the var is initialized as `undefined` and then set to `true` if the condition is met.

Going a bit further, a const wouldn't work here for both of those reasons.  It cannot be initialized with an empty value nor can it's value change.